python/lsst/pipe/analysis/analysis.py

```diff
@@ -436,13 +438,13 @@ def plotAgainstMagAndHist(self, log, filename, stats=None, camera=None, ccdList=
                 axScatter.axvspan(self.magThresholdHigh, axScatter.get_xlim()[1], facecolor="k",
                                   edgecolor="none", alpha=0.10)
                 # compute running stats (just for plotting)
-                if self.calibUsedOnly == 0 and plotRunStats:
+                if plotRunStats:
                     belowThresh = data.mag < magMax  # set lower if you want to truncate plotted running stats
                     numHist, dataHist = np.histogram(data.mag[belowThresh], bins=len(xSyBins))
                     # Only plot running stats if there are a significant number of data points per bin.
-                    # Computed here as the mean number in the brightest 20% of the bins which we require
+                    # Computed here as the mean number in the brightest 10-30% of the bins which we require
```

Comment:
This comment could be clearer, why the change from 20% to 10 - 30%?

Reply:
Amended the comment.